### PR TITLE
:sparkles: [PRTL-450] match chart colors

### PR DESCRIPTION
**Pull request by cy**
screen shot 2016-12-09 at 5 13 00 pm

### CODE REVIEW

**Comment by cy** [comment]
hold off reviewing this for a bit, want to try one more thing with the colors

**Comment by cy** [comment]
ready for review. includes this

> Hi Christine,
> Could you please replace the project code by the full project name in your donut chart's tooltips?
> Thanks
> Vincent

File: app/react-components/charts/StackedBarChart.js [hunk 1]
```diff
@@ -108,9 +107,9 @@ const drawChart = ({ data, yAxis, styles, width, height }) => {
 };
 
 let StackedBarChart = (() => {
-  return ({ data, yAxis, styles, width = 450, height = 200 }) => {
+  return ({ data, yAxis, styles, colors, width = 450, height = 200 }) => {
     return Object.keys(data).length ?
-      drawChart({ data: data, yAxis: yAxis, styles: styles, width: width, height: height }) :
+      drawChart({ data: data, yAxis: yAxis, styles: styles, colors: colors, width: width, height: height }) :
```
Comment (alex-wilmer) [inline app/react-components/charts/StackedBarChart.js]: why not shorthands here?

Comment (alex-wilmer) [inline app/react-components/charts/StackedBarChart.js]: if https://github.com/NCI-GDC/portal-ui/pull/1178 can get prioritized then shouldn't end up with these inconsistencies :)

Comment (cy) [inline app/react-components/charts/StackedBarChart.js]: updated

**Comment by shanewilson** [comment]
@alex-wilmer

**Comment by cy** [comment]
i'll merge after changing project_id to project name in stacked bar chart to match double ring chart

**Pull request merged**